deepsensor/model/model.py

```diff
@@ -1084,13 +1083,47 @@ def stitch_clipped_predictions(
                         )
 
                         patches_clipped[var_name].append(patch_clip)
-
-            combined = {
-                var_name: xr.combine_by_coords(patches, compat="no_conflicts")
-                for var_name, patches in patches_clipped.items()
-            }
-
-            return combined
+            
+            # Create blank prediction dataframe. 
+            patchwise_pred_copy = copy.deepcopy(patches_clipped)
+
+            # Generate new blank DeepSensor.prediction object with same extent and coordinate system as X_t.
```

Comment:
This comment isn't accurate since this process creates a dictionary, not a `DeepSensor.model.pred.Prediction` - do we need to create a `Prediction` object here? This is something that's also described on line 1220 - although that section also doesn't create a `Prediction` object.

These are things I need to address for the patchwise PR anyway so I'm not too concerned about fixing them in this one, just trying to make sure I understand the comments in the code.